core/src/main/java/io/questdb/cairo/mv/MatViewGraph.java

```diff
@@ -154,6 +154,27 @@ public void removeView(TableToken matViewToken) {
         }
     }
 
+    public void updateToken(TableToken updatedToken) {
+        final MatViewDefinition viewDefinition = definitionByTableDirName.get(updatedToken.getDirName());
+        if (viewDefinition != null) {
+            viewDefinition.updateToken(updatedToken);
+            MatViewDependencyList viewList = dependentViewsByTableName.get(viewDefinition.getBaseTableName());
+            if (viewList != null) {
+                var matViews = viewList.lockForWrite();
+                try {
+                    for (int i = 0, n = matViews.size(); i < n; i++) {
+                        if (matViews.get(i).getDirName().equals(updatedToken.getDirName())) {
```

Comment:
should this be ignoring case?

Reply:
I don't see how we can have table dirName flowing around in different cases, but just in case, I've changed it in [17981c3](https://github.com/questdb/questdb/pull/5679/commits/17981c3673618ca895a94b02a4332ce245704419)

Reply:
A lot of code already assumes same case in table dir names. Those names are assumed to be normalized while table names aren't.